packages/nuxt/src/app/composables/cookie.ts

```diff
@@ -26,7 +26,7 @@ const CookieDefaults = {
   path: '/',
   watch: true,
   decode: val => destr(decodeURIComponent(val)),
-  encode: val => encodeURIComponent(typeof val === 'string' ? val : JSON.stringify(val))
+  encode: val => encodeURIComponent(val)
```

Comment:
Let's avoid changing this behaviour and just handle with `klona` or etc. below

```suggestion
  encode: val => encodeURIComponent(typeof val === 'string' ? val : JSON.stringify(val))
```

Reply:
Gotcha